packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -48,6 +48,16 @@ class CodeView extends StatefulWidget {
   static double get rowHeight => scaleByFontFactor(20.0);
   static double get assumedCharacterWidth => scaleByFontFactor(16.0);
 
+  static num calculateLineWidth(textSpan) {
```

Comment:
move this to ui/util.dart

Reply:
give textSpan a type/

Reply:
Also use double as the return type instead of num

Reply:
Done, thanks!